CMakeLists.txt

```diff
@@ -225,10 +226,10 @@ if(AVX2)
   set(SIXTRACK_FEAT_CR "${SIXTRACK_FEAT_CR}_avx2")
   set(SIXTRACK_FEAT_DA "${SIXTRACK_FEAT_DA}_avx2")
 endif(AVX2)
-if(AVX-512)
+if(AVX512 OR AVX512_SKYLAKE)
```

Comment:
Are these flags compatible? Is it OK to activate both?

Reply:
They are just meta flags, so SKYLAKE includes the other one. I can double-check both compilers, but in principle all you get is duplicate flags in the list.

Reply:
gfortran has no complaint, but ifort will override the skylake one with the common one with a warning. I'll add checks to make sure the input makes sense.